core/src/env/call/execution_input.rs

```diff
@@ -15,6 +15,7 @@
 use crate::env::call::Selector;
 
 /// The input data for a smart contract execution.
+#[derive(Debug)]
```

Comment:
Intentional Debug derive here and below?

Reply:
Yes actually `Debug` derives are normally not bad since they allow for debugging and typically do not slow down compilation too much. For our release builds they are compiled out of the resulting Wasm file.